src/plugins/plugin-events.ts

```diff
@@ -19,12 +19,24 @@ const EventsPlugin: Plugin<EventsAPI & PrivateEventsAPI> = {
     return api._obj.isUsed();
   },
 
+  fnWrap: (fn) => (G, ctx, ...args) => {
```

Comment:
Could we add a comment here explaining what exactly this does? It probably won’t be very obvious when we come across it in the future.